Essentia/UserCases/Login/Controllers/WelcomeViewController.swift

```diff
@@ -34,19 +27,27 @@ class WelcomeViewController: BaseViewController, ImportAccountDelegate, SelectAc
     
     override func viewDidLayoutSubviews() {
         super.viewDidLayoutSubviews()
-        applyDesign()
+        self.tableAdapter.hardReload(state)
     }
     
-    func applyDesign() {
-        restoreButton.setTitle(LS("Welcome.Restore"), for: .normal)
-        title1Label.text = LS("Welcome.Title1")
-        title2Label.text = LS("Welcome.Title2")
-        if (inject() as ViewUserStorageServiceInterface).users.isEmpty {
-            enterButton.setTitle(LS("Welcome.Start"), for: .normal)
-        } else {
-            enterButton.setTitle(LS("Welcome.Enter"), for: .normal)
-        }
-        
+    // MARK: - State
+    override var state: [TableComponent] {
+        return [
+            .empty(height: 30, background: colorProvider.settingsCellsBackround),
+            .rightBorderedButton(title: LS("Welcome.Restore"), action: restoreAction,
+                                 borderColor: colorProvider.borderedButtonBorderColor.cgColor, borderWidth: 2),
+            .empty(height: 88, background: colorProvider.settingsCellsBackround),
+            .titleWithFontAligment(font: AppFont.regular.withSize(36), title: LS("Welcome.Title1"), aligment: .left, color: colorProvider.appTitleColor),
+            .titleWithFontAligment(font: AppFont.bold.withSize(36), title: LS("Welcome.Title2"), aligment: .left, color: colorProvider.appTitleColor),
+            .calculatbleSpace(background: .clear),
+            .centeredButton(title: (inject() as ViewUserStorageServiceInterface).users.isEmpty ? LS("Welcome.Start") : LS("Welcome.Enter"),
+                            isEnable: true,
+                            action: enterAction,
+                            background: colorProvider.settingsCellsBackround)]
+            + termsState
```

Comment:
Do not separate state if is not necessarily. Put `termsState` to `state`. Attributed text put to private variable.

Reply:
fixed, ef08e75